subsys/bluetooth/host/direction.c

```diff
@@ -0,0 +1,132 @@
+/*
+ * Copyright (c) 2020 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+#include <stdint.h>
+#include <assert.h>
+
+#include <bluetooth/hci.h>
+#include <bluetooth/conn.h>
+#include <sys/byteorder.h>
+
+#include "conn_internal.h"
+
+#define BT_DBG_ENABLED IS_ENABLED(CONFIG_BT_DEBUG_HCI_CORE)
+#define LOG_MODULE_NAME bt_df
+#include "common/log.h"
+
+/* @brief Function provides information about DF antennae numer and
+ *	  controller capabilities related with Constant Tone Extenstion.
+ *
+ * @param[out] switch_sample_rates	Optional switchin and sampling rates.
+ * @param[out] num_ant			Antennae number.
+ * @param[out] max_switch_pattern_len	Maximum supported antennae switchin
+ *					paterns length.
+ * @param[out] max_CTE_len		Maximum length of CTE in 8[us] units.
```

Comment:
Same thing here with tabs and alignment - only use spaces

Reply:
spaces inserted instead of tabs